pkg/netutil/netutil_test.go

```diff
@@ -0,0 +1,17 @@
+package netutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPublicIP(t *testing.T) {
```

Comment:
Currently we use http.Client to request the public ip,  we'd mock the `ifconfig.me` to verify the function.

Reply:
done